pkg/apiserver/plugins.go

```diff
@@ -59,7 +57,6 @@ var (
 		deletionconfirmation.PluginName,            // DeletionConfirmation
 		openidconnectpreset.PluginName,             // OpenIDConnectPreset
 		clusteropenidconnectpreset.PluginName,      // ClusterOpenIDConnectPreset
-		shootstatedeletionvalidator.PluginName,     // ShootStateDeletionValidator
```

Comment:
Can you mention the removal of the `ShootStateDeletionValidator` admission plugin in the release note?
If someone enables/disables the plugin explicitly, g-api will refuse to start.

Reply:
done